### PR TITLE
Add can_interface specifier and can configure skip

### PR DESCRIPTION
**Pull request by jr1221**
- Add an argument (arg 3) to specify the can interface (required argument). This is so I can change the can interface port for socketcand testing on buildroot.

- Add an argument (arg 4) to skip can interface configuration.  When `skip_can_configure` is specified, it will not run the configure_can function.  This is because buildroot has its own `ip` command that should be used, and sudo doesnt even exist.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -85,14 +84,22 @@ fn parse_args() -> (String, Box<dyn Client + 'static + Send>) {
     }
     let client_type = &args[1];
     let path = &args[2];
+    let can_interface = &args[3];
```
Comment (Peyton-McKee) [inline src/main.rs:87]: You need to validate that you have three args or this will fatal error

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
Looks good other than that one access problem

File: src/main.rs (hunk 1 above)
Comment (jr1221) [reply src/main.rs:88]: Alright, should be fixed.  I added a part that warns before exiting that not enough arguments are specified.

File: src/main.rs [hunk 2]
```diff
@@ -76,23 +75,32 @@ fn read_can(mut publisher: Box<dyn Client + Send>) {
  * Parses the command line arguments.
  * Returns the client type and the path to connect to.
  */
-fn parse_args() -> (String, Box<dyn Client + 'static + Send>) {
+fn parse_args() -> (String, Box<dyn Client + 'static + Send>, String, bool) {
     let args: Vec<String> = env::args().collect();
     println!("{:?}", args);
-    if args.len() < 2 {
-        println!("Please provide a client type");
+    if args.len() < 4 {
```
Comment (Peyton-McKee) [inline src/main.rs:81]: instead of requiring a can interface just make the default one “can0”

**Review by Peyton-McKee — APPROVED** [review]
looks good

**Pull request merged**